PyErr_Occurred,FUNC_0
PyObject_CallMethod,FUNC_1
PyObject_IsTrue,FUNC_2
Py_XDECREF,FUNC_3
fetch_level_from_cache,FUNC_4
level_cache_context,VAR_0
cache_paths,VAR_1
num_cache_paths,VAR_2
key,VAR_3
pk3_path,VAR_4
output,VAR_5
result,VAR_6
